### PR TITLE
New intrinsics

### PR DESCRIPTION
**Pull request by jafri**
Closes issue #70

### CODE REVIEW

File: as-packages/chain/assembly/crypto.ts [hunk 1]
```diff
@@ -114,6 +115,38 @@ export class Checksum512 implements Packer {
     }
 }
 
+export class Checksum1024 implements Packer {
+    data!: u8[];
+
+    pack(): u8[] {
+        return this.data;
```
Comment (learnforpractice) [inline as-packages/chain/assembly/crypto.ts:122]: I think we should return a copy of the data here.

Comment (learnforpractice) [reply as-packages/chain/assembly/crypto.ts:122]: Also for other crypto structs like Checksum160.

**Review by learnforpractice — APPROVED** [review]
Good work, I will approve this PR first.

**Comment by jafri** [comment]
Is the copy just .slice()? Not sure about assemblyscript
________________________________

**Pull request merged**